### PR TITLE
Add a yml file of all Project Jupyter dev repos

### PR DESCRIPTION
**Pull request by willingc**

### CODE REVIEW

File: jupyter-all.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+repos:
+  - ipython/ipython
+  - ipython/ipykernel
+  - ipython/ipyparallel
+  - ipython/ipywidgets
+  - ipython/traitlets
+  - jupyter/dashboards
+  - jupyter/declarativewidgets
+  - jupyter/docker-stacks
```
Comment (minrk) [inline jupyter-all.yml:9]: docker-stacks doesn't do releases

File: jupyter-all.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  - jupyter/jupyter_client
+  - jupyter/jupyter_console
+  - jupyter/jupyter_core
+  - jupyter/kernel_gateway
+  - jupyter/nbconvert
+  - jupyter/nbdime
+  - jupyter/nbformat
+  - jupyter/nbgrader
+  - jupyter/nbviewer
```
Comment (minrk) [inline jupyter-all.yml:18]: same for nbviewer

File: jupyter-all.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  - jupyter/notebook
+  - jupyter/qtconsole
+  - jupyter/tmpnb
```
Comment (minrk) [inline jupyter-all.yml:21]: and tmpnb

File: jupyter-all.yml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  - jupyterlab/jupyterlab
+  - jupyterlab/extension-builder
+  - jupyterlab/extension-cookiecutter-ts
+  - jupyterlab/extension-cookiecutter-js
```
Comment (minrk) [inline jupyter-all.yml:25]: I think these three `extension-` don't do releases either

File: jupyter-all.yml [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  - jupyterhub/kubespawner
+  - jupyterhub/sudospawner
+  - jupyterhub/systemdspawner
+  - jupyterhub/wrapspawner
+  - jupyterhub/jupyterhub-deploy-docker
+  - jupyterhub/jupyterhub-deploy-hpc
+  - jupyterhub/jupyterhub-deploy-teaching
+  - jupyterhub/jupyterhub-carina
+  - jupyterhub/jupyterhub-example-kerberos
+  - jupyterhub/jupyterhub-tutorial
```
Comment (minrk) [inline jupyter-all.yml:44]: These deploy/example repos also don't do releases

File: jupyter-all.yml [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+  - jupyter/notebook
+  - jupyter/qtconsole
+  - jupyter/tmpnb
+  - jupyterlab/jupyterlab
+  - jupyterlab/extension-builder
+  - jupyterlab/extension-cookiecutter-ts
+  - jupyterlab/extension-cookiecutter-js
+  - jupyterlab/jupyterlab_geojson
+  - jupyterlab/jupyterlab_json
+  - jupyterlab/mimerender-cookiecutter
```
Comment (minrk) [inline jupyter-all.yml:28]: I don't think the cookie cutters have releases

**Review by minrk — COMMENTED** [review]
Thanks! This page only makes sense for repos that correspond to packages with releases, which is less than all of these repos. I marked a few that I don't think have releases and thus don't make sense here.

**Comment by willingc** [comment]
Yeah, I was thinking that it might be better to put the logic of whether there was a release in `release-page` app. How about for now that I add a yml file with just those that would populate a release in the report.

**Comment by willingc** [comment]
Added a file `jupyter-releases.yml` that contains only repos that have releases.

**Comment by minrk** [comment]
> Yeah, I was thinking that it might be better to put the logic of whether there was a release in release-page app.

That's a good idea. I'll probably update the app to take orgs as an argument, and check the API for repos with tags.

**Pull request merged**